### PR TITLE
Add script generate-graphext-css

### PR DESCRIPTION
**Pull request by NachoJusticia**

### CODE REVIEW

File: packages/node-build-scripts/package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "test": "exit 0"
   },
   "bin": {
+    "add-js-version": "./add-js-version.sh",
```
Comment (crispamares) [inline packages/node-build-scripts/package.json]: How the add-js-version ends in node_modules/.bin/add-js-version.sh ?  Is it in the yarn install of this package ?

File: packages/node-build-scripts/add-js-version.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# This script adds the Version of the corresponding javascript package in the line 6 of the generated css.
```
Comment (crispamares) [inline packages/node-build-scripts/add-js-version.sh]: I think we should add a disclaimer like: "DISCLAIMER: This only works in Mac OSx"

File: packages/node-build-scripts/package.json (hunk 1 above)
Comment (NachoJusticia) [reply packages/node-build-scripts/package.json]: Yes, the `yarn install` 👍

File: packages/node-build-scripts/add-js-version.sh (hunk 2 above)
Comment (NachoJusticia) [reply packages/node-build-scripts/add-js-version.sh]: After my latest commit now you will get this error:
captura de pantalla 2018-10-17 a las 16 16 52
Is it ok?

Comment (crispamares) [reply packages/node-build-scripts/add-js-version.sh]: Perfect :)

File: packages/node-build-scripts/package.json (hunk 1 above)
Comment (crispamares) [reply packages/node-build-scripts/package.json]: 👍

**Pull request merged**